tools/installbazel/main.sh

```diff
@@ -28,4 +28,16 @@ function install_bazel() {
   apt-get update && apt-get install -y bazel zip unzip
 }
 
-install_bazel
+function install_bazel_aarch64() {
+  BAZELISK_VERSION=v1.19.0
+  apt install wget
+  tmpdir="$(mktemp -t -d bazel_installer_XXXXXX)"
+  trap "rm -rf $tmpdir" EXIT
+  pushd "${tmpdir}"
+  wget "https://github.com/bazelbuild/bazelisk/releases/download/${BAZELISK_VERSION}/bazelisk-linux-arm64"
```

Comment:
we may want to check checksum for the binary

Reply:
I couldn't find a checksum in https://github.com/bazelbuild/bazelisk/releases/tag/v1.19.0.